action.yml

```diff
@@ -176,40 +176,64 @@ runs:
 
     - name: Tar log files (Linux)
       if: inputs.platform == 'ubuntu-latest' && inputs.action == 'stop'
+      env:
+        ROOT_DIR: "/home/runner/.local/share/safe/"
```

Comment:
Hi Roland,

Just a style issue. I would prefer if the trailing slash could be removed from these variables.

I personally think that
```
find "${ROOT_DIR}/node" -iname '*.log*' | tar -zcvf node_log_files.tar.gz --files-from -
```
Is more readable than
```
find "${ROOT_DIR}node" -iname '*.log*' | tar -zcvf node_log_files.tar.gz --files-from -
```

The same would apply to the other relevant variables.

Reply:
Yeah, that makes it even more readable. Updated my PR.